src/clint.py

```diff
@@ -2600,16 +2600,23 @@ def CheckBraces(filename, clean_lines, linenum, error):
         else:
             func_start_linenum = end_linenum + 1
             while not clean_lines.lines[func_start_linenum] == '{':
-                if not Match(r'^(?:\s*\b(?:FUNC_ATTR|REAL_FATTR)_\w+\b(?:\(\d+(, \d+)*\))?)+$',
-                             clean_lines.lines[func_start_linenum]):
+                attrline = Match(r'^(\s*)(?:\b(?:FUNC_ATTR|FUNC_API|REAL_FATTR)_\w+\b(?:\(\d+(, \d+)*\))?\s*)+$',
```

Comment:
I didn't add `\b`, but I'll be happy to remove it if it is unnecessary (both of them?)